iodata/test/test_orca.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# IODATA is an input and output module for quantum chemistry.
+#
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+# pragma pylint: disable=invalid-name,no-member
+"""Test iodata.orca module."""
+
+import numpy as np
+
+from ..iodata import IOData
+from ..utils import angstrom
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
+
+
+def test_load_water_number():
+    # test if IOData has atomic numbers
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def test_load_water_element():
+    # test if IOData has atomic symbols
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def test_load_scf_energy():
+    # test if IOData has the correct energy
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def check_water(mol):
+    """Checks if atomic numbers and coordinates obtained from orca out file are correct.
+
+        Parameters
+        ----------
+        mol : IOData
+            IOdata dictionary.
+
+        Returns
```

Comment:
No need to document return if nothing is returned.

Reply:
deleted Return